src/auth/auth.service.ts

```diff
@@ -0,0 +1,67 @@
+import { HttpStatus, HttpException, Injectable } from '@nestjs/common'
+@Injectable()
+export class AuthService {
+  async getOtp(phoneNumber: string) {
+    // try {
+    //   const sdk = require('api')('@thaibulksms/v1.0#3s3hunt2tktwn9w2l')
+
+    //   const res = await sdk.post(
+    //     '/v2/otp/request',
+    //     {
+    //       key: process.env.OTP_KEY || '1727804793031446',
+    //       secret: process.env.OTP_SECRETKEY || 'b97cef0144f23791501a33afe87a23f5',
+    //       msisdn: phoneNumber,
+    //     },
+    //     { Accept: 'application/json' }
+    //   )
+    //   return res
+    // } catch (err) {
+    //   console.error(err)
+    //   throw new HttpException('Failed Requesting OTP', HttpStatus.BAD_REQUEST)
+    // }
```

Comment:
are u gonna use the code that are commended again? if not please delete unused code

Reply:
yes this is when i want to connect to the actual third party company providing otp service

Reply:
oh my bad